File: src/main/java/ru/citeck/idea/completion/js/JsCompletionContributor.java
package ru.citeck.idea.completion.js;

import com.intellij.lang.Language;
import com.intellij.lang.javascript.JavascriptLanguage;
import ru.citeck.idea.completion.CompletionContributor;

public class JsCompletionContributor extends CompletionContributor {
    @Override
    public Language getLanguage() {
        return JavascriptLanguage.INSTANCE;
    }
}

File: src/main/kotlin/ru/citeck/idea/completion/js/JsCompletionContributor.kt
package ru.citeck.idea.completion.js

import com.intellij.lang.Language
import com.intellij.lang.javascript.JavascriptLanguage
import ru.citeck.idea.completion.js.contributor.CompletionContributor

class JsCompletionContributor : CompletionContributor() {
    override fun getLanguage(): Language {
        return JavascriptLanguage
    }
}
